cmd/compose/compose.go

```diff
@@ -301,6 +302,7 @@ func RootCommand(backend api.Service) *cobra.Command {
 		imagesCommand(&opts, backend),
 		versionCommand(),
 		buildCommand(&opts, backend),
+		tagCommand(&opts, backend),
```

Comment:
After reviewing [the ticket](https://github.com/docker/compose/issues/9051) it seems really needed use case, instead of context switching to plain docker for tagging and pushing. GOOD feature to have 👍🏼

Reply:
it's time to not wait